drivers/lora/sx126x_stm32wl.c

```diff
@@ -45,6 +50,71 @@ void sx126x_dio1_irq_disable(struct sx126x_data *dev_data)
 	irq_disable(DT_INST_IRQN(0));
 }
 
+void sx126x_set_tx_params(int8_t power, RadioRampTimes_t ramp_time)
+{
+	uint8_t buf[2];
+
+	if (pa_output == RFO_LP) {
+		const int8_t max_power = DT_INST_PROP(0, rfo_lp_max_power);
+
+		if (power > max_power) {
+			power = max_power;
+		}
+		if (max_power == 14) {
+			SX126xSetPaConfig(0x04, 0x00, 0x01, 0x01);
+			power = 14 - (max_power - power);
+		} else if (max_power == 10) {
+			SX126xSetPaConfig(0x01, 0x00, 0x01, 0x01);
+			power = 13 - (max_power - power);
+		} else { /* default +15 dBm */
```

Comment:
This should probably not be considered the default, see also other comment.

Reply:
Agreed, see other comment, will update.